.travis.yml

```diff
@@ -27,4 +33,7 @@ branches:
 before_install: 
   - "npm install -g grunt-cli bower@1.3.9"
 
-script: "mvn verify javadoc:jar"
+script: 
+- mvn verify javadoc:jar
+- mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent package sonar:sonar
```

Comment:
Note I didn't test this as I can't execute it on Travis. The addons part should be sane and I had to slightly modify the script part. Not hugely familiar with Travis so I welcome pointers here :)

Reply:
@lgriffin  you can see what @travis-ci did in the [build log](https://s3.amazonaws.com/archive.travis-ci.org/jobs/284214927/log.txt?X-Amz-Expires=30&X-Amz-Date=20171006T164047Z&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJRYRXRSVGNKPKO5A/20171006/us-east-1/s3/aws4_request&X-Amz-SignedHeaders=host&X-Amz-Signature=fe7d22a64c661e607f4ced468165410da91ddb06e6ae674d5144ed1cb8e295e6) :wink:

Reply:
Thanks for the link I missed it earlier, has expired now 🙈  Will know for my next PR!